SoftLayer/CLI/hardware/credentials.py

```diff
@@ -22,10 +22,13 @@ def cli(env, identifier):
                                      'hardware')
     instance = manager.get_hardware(hardware_id)
 
-    table = formatting.Table(['username', 'password'])
+    table = formatting.Table(['username', 'password', 'Software', 'Version'])
```

Comment:
The new fields start with capital letters, the fields like, username and password, could also use the same style